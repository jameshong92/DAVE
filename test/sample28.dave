// test table access
int main() {
    rec r1 = new rec(age: 22, name:"Yang", GPA:3.50); 
    rec r2 = new rec(name:"James", age:23, GPA:3.88);
    rec r3 = new rec(age:24, GPA:3.80, name:"Min Woo");
    tbl test = new tbl([r1, r2, r3]);
    print(test);
    tbl test1 = access(test, 1, 3);
    print(test1);
    tbl test2 = access(test, 1, 100);
    print(test2);
}