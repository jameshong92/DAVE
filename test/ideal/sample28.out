field:
num	
1	
3	
2	
7	
6	

min:
1
max:
7
mean:
3.8
